corfudb-tools/README

```diff
@@ -30,7 +30,7 @@ org.corfudb.browser.CorfuStoreBrowserMain
 
 2. listTables to list all table names in a given namespace(all namespaces if the
 namespace param is null)-
-java -cp "corfudb-tools-0.4.2.1-SNAPSHOT-shaded.jar" --Dlogback.configurationFile=logback.prod.xml
+java -cp "corfudb-tools-0.0.0.0-SNAPSHOT-shaded.jar" --Dlogback.configurationFile=logback.prod.xml
```

Comment:
Just curious why 0.0.0 instead of 4.2.2 ?

Reply:
This is just a README file, so letting users substitute the actual jar name with its correct path is okay.

Making version `0.0.0.0` helps us avoid these changes in all such README files for every version upgrade.